src/lib.rs

```diff
@@ -115,7 +115,7 @@ macro_rules! printfl {
 extern crate time;
 mod tty;
 mod pb;
-mod multi;
+pub mod multi;
 pub use pb::{ProgressBar, Units};
 pub use multi::MultiBar;
```

Comment:
Can you expose  like this: `pub use multi::{MultiBar, Pipe}`? this way it will be easy to use.

Follow your example:
```rust
use pbr::{MultiBar, ProgressBar, Pipe};

// ...

let mut pb: ProgressBar<Pipe>;
```

Thanks